frame/contracts/src/benchmarking/mod.rs

```diff
@@ -2481,6 +2481,22 @@ benchmarks! {
 		sbox.invoke();
 	}
 
+	// w_per_local = w_bench
+	call_per_local {
+		let r in 0 .. INSTR_BENCHMARK_BATCHES;
+		let max_locals = T::Schedule::get().limits.locals;
+		let mut call_body = body::plain(vec![
+				Instruction::End,
+		]);
+		body::inject_locals(&mut call_body, max_locals);
+		let mut sbox = Sandbox::from(&WasmModule::<T>::from(ModuleDefinition {
+			call_body: Some(call_body),
+			.. Default::default()
+		}));
+	}: {
+		sbox.invoke();
+	}
```

Comment:
We need to find out the cost per instruction. Hence we need a component that allows us to do the linear regression over the number of locals. We can't do `r` at the same time as those influence each other.

```suggestion
	// w_per_local = w_bench
	call_per_local {
		let l in 0 .. T::Schedule::get().limits.locals;
		let mut call_body = body::plain(vec![
				Instruction::End,
		]);
		body::inject_locals(&mut call_body, l);
		let mut sbox = Sandbox::from(&WasmModule::<T>::from(ModuleDefinition {
			call_body: Some(call_body),
			.. Default::default()
		}));
	}: {
		sbox.invoke();
	}
```